### PR TITLE
Babel polyfill

### PR DESCRIPTION
**Pull request by ekryski**

### CODE REVIEW

**Comment by daffl** [comment]
Oh, I just squash merged the code coverage commit into master.

**Comment by ekryski** [comment]
Yup wasn't done. Usually that's why the one doing the PR should merge after review. No worries. Just resolving conflicts right now.

File: app/index.js [hunk 1]
```diff
@@ -60,14 +60,16 @@ module.exports = generators.Base.extend({
...
     this.npmInstall([
-      'babel-core@^6.0.0',
-      'babel-cli@^6.0.0',
+      'babel-core@^6.17.0',
+      'babel-cli@^6.16.0',
+      'babel-polyfill@6.16.0',
       'babel-preset-es2015@^6.0.0',
       'babel-plugin-add-module-exports',
       'semistandard',
+      'eslint-if-supported',
```
Comment (ekryski) [inline app/index.js]: okay. Let me try without.

File: app/templates/index.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
```
Comment (daffl) [inline app/templates/index.js]: Feathers core already includes this. We may only have to add it in the plugin tests.

Comment (ekryski) [inline app/templates/index.js]: So why did we add it to feathers-permissions?

File: app/index.js (hunk 1 above)
Comment (ekryski) [inline app/index.js]: Yup works without. 👍

File: app/templates/index.js (hunk 2 above)
Comment (ekryski) [inline app/templates/index.js]: Wow I am smoking crack. We did not add it to feathers-permissions. Sorry too much in flight yesterday. Will remove and ship.

**Comment by ekryski** [comment]
Published v0.9.1 with these changes and #35

**Pull request merged**